en/04_Changelogs/4.13.0.md

```diff
@@ -15,6 +15,10 @@ title: 4.13.0 (unreleased)
 
 The following API have been deprecated in this minor release, and will be removed in a future major release. For API with a replacement, we recommend you update your code to use the replacement as part of your upgrade to Silverstripe CMS 4.13.0.
 
+#### cwp/cwp
+
+Deprecated class [`CWP\CWP\Extensions\CwpSiteSummaryExtension`](api:CWP\CWP\Extensions\CwpSiteSummaryExtension) Will be removed without equivalent functionality to replace it
```

Comment:
```suggestion
- Deprecated class [`CWP\CWP\Extensions\CwpSiteSummaryExtension`](api:CWP\CWP\Extensions\CwpSiteSummaryExtension) Will be removed without equivalent functionality to replace it
```
I'm converting these to lists as part of the beta release process (they should have been lists to start with - oops). This PR has missed the window for being part of that conversion so please update in PR.

Reply:
Updated